base_operations.yaml

```diff
@@ -331,14 +331,11 @@ components:
       type: object
       properties:
         metadata:
-          type: object
-          additionalProperties: true
+          type: array
```

Comment:
Does it make sense to improve the types for `metadata` and `results`?

E.g. by using the `base_metadata` schema?

https://github.com/seatable/openapi/blob/4f8591404f5edbdcbefcdd75b91884371cbee5b6/base_operations.yaml#L351-L361

Reply:
Hi， The metadata described in base_metadata is not same as the metadata returned by sql query. Please ignore it and we will remove it later.